Tutorials/Editors-Manual/Working-with-Content/Rich-Text-Editor/index.md

```diff
@@ -213,12 +216,14 @@ The macros available to you differ from system to system so please see your syst
 2. Click the ***Insert Macro*** button from the toolbar to open the macro properties dialog box.
 3. Select a macro to insert from the Choose a macro drop-down list.
 4. Configure the macro as required.
-5. Click ***Select*** to insert the macro.
+5. Click ***Submit*** to insert the macro.
 6. The macro is displayed in the editor with a yellow dotted border.
 
-![macroPicker.jpg](images/macroPicker.jpg)
+![macroPicker.jpg](images/Macro-picker-button.png)
```

Comment:
In this image, the "Insert Form" macro has already been selected - is that intentional?
As I read the steps, this image is supposed to show me how to pick a macro, right? Or perhaps I'm missing something!